DATASTD-2159 Add Path Domain Documentation To docs.edfi (#278)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: dataStandard_versioned_docs/version-4/model-reference/educator-preparation-data-model-domain/overview.md

    Candidate) can be associated with a credential.
* An expansion to the Survey entity and related entities to support survey
    requirements that are a cornerstone of most EPP implementations.
* The Student Path is a community extension that tracks the progress of an individual
    from being a student to becoming an educator. This domain tracks various
    assessments , fieldwork, and certifications from third party and state education

>     agencies. The [Student Path Domain](student-path.md) community extension can be reviewed in greater detail by following the link.

Reviewer: Links are self-explanatory, not need to tell people.

```suggestion
    agencies. See the [Student Path Domain](student-path.md) community extension for more information.
```

To be honest with myself, that does not look very different on the surface. Still giving someone an instruction. Text is a little shorter though, and the words have more semantic meaning.
